WordPress/src/main/java/org/wordpress/android/ui/comments/CommentDetailFragment.java

```diff
@@ -420,6 +433,13 @@ private void setNoteWithNoteId(String noteId) {
         setRemoteBlogId(note.getSiteId());
     }
 
+    private void setIdForFragmentContainer(int id){
```

Comment:
What do you think if we remove this method and move the assignment `mIdForFragmentContainer = id;` in `newInstance` ? Since it's called only there, and the name of this method is quite similar to ` setIdForCommentContainer` confusing me a bit at the beginning of the review.